open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/view-generator-connectors/virtualization-solution-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/viewgenerator/virtulaizationsolutionconnector/derby/DerbyConnector.java

```diff
@@ -0,0 +1,81 @@
+package org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.derby;
+
+import org.odpi.openmetadata.frameworks.connectors.ConnectorBase;
+import org.odpi.openmetadata.frameworks.connectors.properties.AdditionalProperties;
+import org.odpi.openmetadata.frameworks.connectors.properties.ConnectionProperties;
+import org.odpi.openmetadata.frameworks.connectors.properties.EndpointProperties;
+import org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.VirtualizationSolutionConnectorBase;
```

Comment:
Spelling mistake in package name virtulaizationsolutionconnector.
This part of the package name seems unnecessary - the connector is a View Generation connector so suggests that a package name of 

```
org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.derby
```
is sufficient